packages/fast-components-react-base/src/slider/slider.tsx

```diff
@@ -66,6 +67,7 @@ class Slider extends Foundation<SliderHandledProps, SliderUnhandledProps, Slider
     private static minIncrementDelay: number = 100;
     private static incrementAcceleration: number = 50;
     private static rolePropName: string = "role";
+    private static dirPropName: string = "dir";
```

Comment:
I'm realizing that "dir" and "role" should exist in the `fast-web-utilities` package, not as part of a single class.

@chrisdholt @nicholasrice do we want to provide an exported set of attribute strings for these purposes?

Reply:
we do export a `Direction` object.

Reply:
I added a string for 'dir' called "DirectionAttributeName" to utilities.  Does that fit with your thoughts?